catalog/imaging/imaging-transformer-nitf/src/main/java/org/codice/alliance/transformer/nitf/image/ImageAttribute.java

```diff
@@ -149,10 +151,17 @@
   public static final String TARGET_IDENTIFIER = PREFIX + "target-identifier";
 
   /*
-   * Normalized attributes. These taxonomy terms will be duplicated by `ext.nitf.image.*` when
-   * appropriate
+   * Normalized attributes. These taxonomy terms will be duplicated by `ext.nitf.image.*` when appropriate.
    */
 
+  public static final ImageAttribute MISSION_ID_ATTRIBUTE =
+      new ImageAttribute(
+          Isr.MISSION_ID,
+          "IID2",
+          ImageSegment::getImageIdentifier2,
+          new IsrAttributes().getAttributeDescriptor(Isr.MISSION_ID),
+          "");
```

Comment:
This `Attribute` was moved here from https://github.com/codice/alliance/blob/1faf1f4dd3bb7262cc4088dc74b6b63b7b059519/catalog/imaging/imaging-transformer-nitf/src/main/java/org/codice/alliance/transformer/nitf/image/NitfImageTransformer.java#L157-L167.

Reply:
@peterhuffer, mostly formatting and typos. @clockard did the initial draft of the mappings, and I double-checked everything as I was working on this PR, so the content of the mappings has already been reviewed by me.

A few other things to review:

- Many rows repeat the same NITF Field because it is mapped to multiple attributes. Does this make sense? Should this be reformatted?
- Check the `Translation` column. Do the notes make sense?
- Does the separation of tables and throwing them all in the appendix make sense? Should I reorganize the rows?
- Is there missing `warning` text or notes that should be added to these tables to provide some context?